src/pages/certification/components/AuditorRunTestForm/utils.ts

```diff
@@ -13,6 +13,7 @@ export const clearPersistentStates = () => {
 
 export const RepoField = {
   label: 'GitHub Repository',
+  placeholder: 'URL entered should be in the format - https://github.com/<username>/<repository> (with an optional trailing backslash).',
```

Comment:
If this is kept as a placeholder, the entire text will not be shown in even 2k screens.
That is mainly why I used a configured a tooltip to the component, to also utilise in case we had larger info to be displayed.

![image](https://github.com/input-output-hk/dapps-certification-web/assets/11058223/b2ab78a0-822c-403e-9da7-6206b472cc4d)

Reply:
Ok! I updated to a tooltip now